A6-client-retries.md

```diff
@@ -96,11 +96,9 @@ Each of these configuration options is detailed in its own section below.
 
 ##### Maximum Number of Retries
 
-`maxRetryAttempts` specifies the maximum number of retry attempts per original RPC.
+`maxAttempts` specifies the maximum number of RPC attempts, including the original request.
 
-gRPC's call deadlines apply to the entire chain of original RPC and retry attempts. For example, if the specified deadline for an RPC is July 23 9:00:00pm PDT the operation will fail after that time regardless of how many retries were configured or attempted.
-
-Note that if `maxRetryAttempts` is set to 3, a total of 4 RPCs could be made against the server (if the first three fail): 1 original RPC and 3 retry attempts.
+gRPC's call deadlines apply to the entire chain of original RPC and retry attempts. For example, if the specified deadline for an RPC is July 23 9:00:00pm PDT the operation will fail after that time regardless of how many attempts were configured or attempted.
```

Comment:
Suggest alternate wording for the first sentence:

gRPC's call deadline applies across all attempts for a given RPC.